### PR TITLE
Rough cut of graph fixes

### PR DESCRIPTION
**Pull request by paulbarclay**
Don’t expect this to work; it’s not been linted, tested, or even compiled. It was all written on a phone, so there will be typos. 

I’d recommend going line-by-line to ensure the adds/deletions make sense

### CODE REVIEW

File: src/mappings/core.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ export function handleTransfer(event: Transfer): void {
 
   // get pair and load contract
   let pair = Pair.load(event.address.toHexString())
-  let pairContract = PairContract.bind(event.address)
+  // let pairContract = PairContract.bind(event.address)
```
Comment (sameepsi) [inline src/mappings/core.ts:49]: @paulbarclay It is used at line #196

File: src/mappings/core.ts [hunk 2]
```diff
@@ -89,7 +89,7 @@ export function handleTransfer(event: Transfer): void {
       transaction.mints = mints.concat([mint.id])
 
       // save entities
-      transaction.save()
+      // transaction.save()
```
Comment (sameepsi) [inline src/mappings/core.ts:92]: @paulbarclay Why you are not saving tx entity?

File: src/mappings/core.ts [hunk 3]
```diff
@@ -117,7 +117,7 @@ export function handleTransfer(event: Transfer): void {
     // against unintended side effects for other code paths.
     burns.push(burn.id)
     transaction.burns = burns
-    transaction.save()
+    // transaction.save()
```
Comment (sameepsi) [inline src/mappings/core.ts:120]: same as above

File: src/mappings/core.ts [hunk 4]
```diff
@@ -172,7 +172,7 @@ export function handleTransfer(event: Transfer): void {
       // side effects for other code paths.
       mints.pop()
       transaction.mints = mints
-      transaction.save()
+      // transaction.save()
```
Comment (sameepsi) [inline src/mappings/core.ts:175]: same here

File: src/mappings/core.ts [hunk 5]
```diff
@@ -188,7 +188,7 @@ export function handleTransfer(event: Transfer): void {
       burns.push(burn.id)
     }
     transaction.burns = burns
-    transaction.save()
+    // transaction.save()
```
Comment (sameepsi) [inline src/mappings/core.ts:191]: same here

File: src/mappings/core.ts [hunk 6]
```diff
@@ -229,7 +229,7 @@ export function handleSync(event: Sync): void {
   if (pair.reserve0.notEqual(ZERO_BD)) pair.token1Price = pair.reserve1.div(pair.reserve0)
   else pair.token1Price = ZERO_BD
 
-  pair.save()
+  // pair.save()
```
Comment (sameepsi) [inline src/mappings/core.ts:232]: same here

File: src/mappings/core.ts (hunk 1 above)
Comment (paulbarclay) [reply src/mappings/core.ts:49]: Right. I re-added the use, but not the contract. I rely on intellisense so much.

File: src/mappings/core.ts [hunk 7]
```diff
@@ -238,8 +238,8 @@ export function handleSync(event: Sync): void {
 
   token0.derivedETH = findEthPerToken(token0 as Token)
   token1.derivedETH = findEthPerToken(token1 as Token)
-  token0.save()
-  token1.save()
+  // token0.save()
```
Comment (sameepsi) [inline src/mappings/core.ts:241]: same here

**Review by sameepsi — APPROVED** [review]

**Pull request merged**